packages/ui-core/src/components/table/ceil.component.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import { StyledCeil } from './table.styles';
+import { CeilType } from '../../types';
+import Tooltip from '../tooltip';
+
+type Props = {
+  children?: CeilType;
+  onClick?: (children: CeilType) => void;
+  tooltipEnabled?: boolean;
+  tooltipContent?: string | React.ReactNode;
+  format?: (children: CeilType) => void;
+};
+
+const tooltipMotion = {
+  initial: { opacity: 0 },
+  animate: { opacity: 1 },
+  exit: { opacity: 0 },
+};
+
+export const Ceil = ({
+  children,
+  onClick,
+  tooltipEnabled = true,
+  tooltipContent = 'Copied!',
+  format,
+}: Props) => {
+  const [tooltip, setTooltip] = useState(false);
+
+  const toogleTooltip = () => {
+    onClick(children);
```

Comment:
I'll change the interface of that component - to be not depend on `children` as this could lead in the future to strange behaviour - if someone wrap <Ceil /> children in additional HTML element.

Please extend interface with dedicated and more precise type for eg.. `value: string | number`.